tests/dtls-server.c

```diff
@@ -324,6 +332,11 @@ main(int argc, char **argv) {
 
   dtls_init();
 
+  if (signal(SIGINT, dtls_handle_signal) == SIG_ERR) {
```

Comment:
As `signal()` is explicitly deprecated, I suggest using `sigaction()` instead.

Reply:
Updated and code pushed